lib/containers/container_images.pm

```diff
@@ -56,9 +56,9 @@ sub build_container_image {
     assert_script_run("$runtime images");
 }
 
-=head2 test_containered_app
+=head2 build_and_run_image
 
- test_containered_app($runtime, [$buildah], $dockerfile, $base);
+ build_and_run_image($runtime, [$buildah], $dockerfile, $base);
```

Comment:
in case @b10n1k  will be convinced we also need to update this line , but ofc. ignore this message if you will revert param removal